governance.md

```diff
@@ -197,56 +218,219 @@ interests include, but are not limited to:
+
+An individual may be the RM as more than one release series at the
+same time.
+
+
+### API consistency Leader
+
+Matplotlib is constantly making small changes to our API: enhancements
+that add new features, bug fixes that unavoidably change behavior, and
+deprecation of inconsistent or undesired functionality.  The API
+Consistency Leader (ACL) is responsible for making sure that these
+incremental changes to the library are done in a coherent and
+consistent manner.
+
+This include checking that:
+
+- new functionality is not duplicating existing functionality
+- deprecations are justified and properly documented
+- new functionality does not "paint us into a corner" for future work
+- new functionality is implemented with an API that is consistent with
+  the existing functions
+
+
+### Reference Documentation Leader
+
+The Matplotlib API reference documentation is split between the docstrings and
+the rst source.  This documentation needs to be complete and accurate as our
+users rely on it as the final authority of what a given method will do (short of
+reading the source).
+
+The Reference Documentation Leader (RDL) is responsible for ensuring that
+the docstrings are:
+
+- correctly formatted and render as intended
+- technically correct
+- complete
 
+In addition to the docstrings the RDL is responsible for the sphinx build
+machinery and our sphinx extensions.
+
+### Narrative Documentation Leader
+
+In addition to the refernce documentation, Matplotlib has narrative documentation.
+This documentation can take the form of short "cookbook" examples that show an
+end-to-end
+
+
+This includes our examples and tutorials that live inside the
+main source repository and longer tutorials that live in other repositories
+in the Matplotlib organization.
+
+The Narrative Documentation Leader is responsible for shepherding all of this
+content including the scope, level, tone, and voice.
+
+### Secretary
+
+- Responsible for ensure that there is an agenda for the weekly meeting at it is
+  followed.
+- Responsible for maintaining the weekly notes.
+- Responsible for ensuring that decisions taken by vote in the SC are properly
+  recorded.
+
+### Community Manager
+
+The true strength of Matplotlib and why it has had such longevity as a project
+is the community of people around the code.  That community needs to be
+maintained.  The Community Manager (CM) is the is a catch-all for several very
+diverse tasks and this role may be split in the future and may want to enlist
+further assistants.
+
+**Communications**
+
+We need to maintain communication channels with the wider world.  To this end
+the CM is responsible for maintaining:
+
+- the [Ann] mailing list
+- The Matplotlib blog
+- Maintaining a "matplotlib update" slide deck
+- Recruiting people to give talks on Matplotlib at conferences / meetups
+- Matplotlib's social media presence
+
+This communication is primarily broadcast out and intended to engage with
+end-users.
```

Comment:
change end -users to broader community. Like a good chunk of our social media engagement is core devs on other projects.